gradle/gradle-enterprise.gradle

```diff
@@ -36,6 +36,14 @@ static java.util.Optional<String> getSetting(String name) {
     return java.util.Optional.ofNullable(sysProp);
 }
 
+static boolean isEnabled(String setting) {
+    if ( System.getenv().hasProperty( setting ) ) {
+        return true
```

Comment:
isn't this going to return "true" even when setting `-DPOPULATE_REMOTE=false` ?

Reply:
Its not meant to be a true/false flag

Reply:
I just had to pass *some* value